core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -1076,6 +1134,10 @@ protected MasterWorkerInfo recordWorkerRegistration(long workerId) {
 
   @Override
   public long getWorkerId(WorkerNetAddress workerNetAddress) {
+    if (mStandbyMasterRpcEnabled && mPrimarySelector.getState() == NodeState.STANDBY) {
```

Comment:
Is this `mStandbyMasterRpcEnabled` condition check redundant?

Reply:
yes. i added this because a bunch of tests use a default STANDBY primary selector. If i don't add this, a lot of tests will break